htdocs/admin/security.php

```diff
@@ -318,7 +325,7 @@
 
 	print '<tr class="oddeven">';
 	print '<td>' . $langs->trans("NbIteConsecutive")."</td>";
-	print '<td colspan="2"><input type="number" value="'.$tabConf[4].'" id="NbIteConsecutive" min="0"></td>';
+	print '<td colspan="2"><input type="number" value="'.$tabConf[4].'" id="NbIteConsecutive" min="1"></td>';
```

Comment:
Why disabling the value "0", "0" should means "this constraint is not checked", shouldn't it ?

Reply:
It's not how it's taken into account in the generation of the password. But, I could adapt this. For me, the fix is the correct solution.

If for you 0 means: "don't take that value into account", then we can change but you should know, that's not the behaviour of the other fields. So it's a change and not a fix then imo

Here my PR is well to fix the page _security.php_ when we selected 0. 
![image](https://user-images.githubusercontent.com/3658458/66298490-2612b280-e8f2-11e9-8961-2185d1684040.png)

Reply:
I'll have a look later to check if it's a lot of changes or not.

Reply:
It was my understanding but it does not means this was what the developer expected to do. Because, this is a common behaviour in dolibarr to be able to use 0 to disable a setup, i also made the change after your patch. So 0 is also accepted now too.

Reply:
And is it working? Because then I guess  the initial issue#12041 is still there isn't it ?

Reply:
@eldy I confirm, by your change the issue is not fixed. I'll push a new code taking your idea into account.

Cheers
L.